clients/src/main/java/org/apache/kafka/common/protocol/ApiKeys.java

```diff
@@ -110,6 +110,14 @@
     LIST_TRANSACTIONS(ApiMessageType.LIST_TRANSACTIONS),
     ALLOCATE_PRODUCER_IDS(ApiMessageType.ALLOCATE_PRODUCER_IDS, true, true);
 
+    /**
+     * ALTER_ISR was the old name for ALTER_PARTITION.
+     *
+     * @deprecated since 3.2.0. Use {@link #ALTER_PARTITION} instead
+     */
+    @Deprecated
```

Comment:
Okay. I wasn't sure if the symbols in `common/protocol/ApiKeys.java` were public. I am okay keeping it as it is a light way of keeping backward compatibility just in case external code is using this `enum`.

Reply:
I think I'd just get rid of it. Deprecating it sort of sends the wrong signal that it is a public API, when it is definitely not.

Reply:
Done.